src/en/guarantees.md

```diff
@@ -38,3 +38,38 @@ However, the language does not prevent
 
 * resource leaks (memory, IO, ...),
 * numeric overflows.
+
+## LLVM Tier guarantees
```

Comment:
I am not sure about putting LLVM everywhere in the document, when the official Rust doc do not cite once LLVM. And personally I cannot find a current statement from the LLVM community about them either.